crawl-ref/source/dat/des/variable/mini_features.des

```diff
@@ -2758,6 +2758,101 @@ MAP
 ..............
 ENDMAP
 
+NAME: chequers_decorated_corner
+TAGS: decor extra
+DEPTH: D, Depths
+MAP
+xxxxxxxxxx
+x.......G@
+x.TG.....@
+x.Gxxxxxxx
+x..x
+xG.x
+x@@x
+ENDMAP
+
+NAME: chequers_jeans
+TAGS: decor extra
+DEPTH: D, Depths
+SHUFFLE: G$
+MAP
+....@....
+.xxxxxxx.
+.xxxGxxx.
+.xxx.xxx.
+.xxx.xxx.
+.xxx.xxx.
+.xx...xx.
+.xx...xx.
+.xx...xx.
+.xx...xx.
+.xx...xx.
+.........
+ENDMAP
+
+NAME: chequers_mini_garden
+TAGS: decor extra
+DEPTH: D
+COLOUR: g = green
+FTILE:  g = floor_grass
+SUBST:  g = . t
+FTILE:  . = floor_dirt / floor_grass / none
+COLOUR: . = brown / green / none
+MAP
+....
+.gg.
+.gg.
+....
+ENDMAP
+
+NAME: chequers_garden
+TAGS: decor extra
+DEPTH: D
+COLOUR: g = green
+FTILE:  g = floor_grass
+SUBST:  g = . t
+FTILE:  . = floor_dirt / floor_grass / none
+COLOUR: . = brown / green / none
+MAP
+........
+........
+..gggg..
+..gg.g..
+..g.gg..
+..gggg..
+........
+........
+ENDMAP
+
+NAME: chequers_avenue
+TAGS: decor extra
+DEPTH: D
+SUBST: t = G:3 t
+SHUFFLE: tG
+MAP
+xxxxxxxxxxxxxxxxxxxxxxxxx
+.t.t.t.t.t.t.t.t.t.t.t.t.
+@.......................@
+.t.t.t.t.t.t.t.t.t.t.t.t.
+xxxxxxxxxxxxxxxxxxxxxxxxx
+ENDMAP
+
+NAME:   chequers_arboreum
+TAGS:   decor extra no_item_gen no_monster_gen
+PLACE:  D:1
+DEPTH:  D
+SUBST:  g = t.
+COLOUR: . = green, g = green
```

Comment:
So here we're turning each `g` into either a tree with a normal floor colour/tile, or a floor tile that's green/grass? If `.` is supposed to be the same as `g` after the transform, why not just use `.` instead of `g` to begin with? That is, replace all the `g`s in the vault with `.`, and then

    SUBST: . = t.
    COLOUR: . = green
    FTILE: . = floor_grass

Reply:
I agree that you don't need an intermediate glyph. Also, he will need to have `FTILE` set on both `t` and `.`, contrary to your code above, since otherwise trees will have a normal dungeon floor tile, which would look odd. Alex can simplify his original `FTILE` statement though:
```
FTILE: t. = floor_grass
```
instead of putting two statements setting the same tile.

Reply:
Okay. I wasn't sure if setting the tile for a tree would cause it to no longer look like a tree.